elm-pages develop throws error in v1.2.8

Hello Dillon!

I saw that you had closed an issue relating to the missing content flash and decided to bump my elm-pages version from 1.2.5 (the version hardcoded into the elm-pages-starter project) to the latest and greatest.

When I ran npm start I got the following error.

---

> elm-pages develop

module.js:550
    throw err;
    ^

Error: Cannot find module '../node-elm-compiler/index.js'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/myproject/node_modules/elm-pages/generator/src/compile-elm.js:1:91)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/myproject/node_modules/elm-pages/generator/src/generate-elm-stuff.js:2:16)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! Exit status 1

---

Not a huge problem. I dropped back to 1.2.7 and everything seems to work fine there. Just thought you should have the heads up.

Thank again for all your work!

--- a/package.json
+++ b/package.json
@@ -66,6 +66,7 @@
   },
   "files": [
     "index.js",
+    "generator/node-elm-compiler/",
     "generator/src/"
   ],
   "bin": {

